pkg/config/config.go

```diff
@@ -1143,6 +1132,24 @@ func findUnknownEnvVars(config Config) []string {
 	return unknownVars
 }
 
+func useHostEtc(config Config) {
+	if IsContainerized() && pathExists("/host/etc") {
+		if !config.GetBool("ignore_host_etc") {
```

Comment:
We could consider doing this config first, instead of after a `IsContainerized()` or `pathExists` call, WDTY?

Reply:
I wanted to print a debug message when `ignore_host_etc` is enabled and has an impact (ie: containerized+/host/etc)